frontend/src/components/eMIB/EmibTabs.jsx

```diff
@@ -40,7 +40,19 @@ class EmibTabs extends Component {
     ];
     return (
       <div style={customStyles.container}>
-        <TabNavigation tabSpecs={TABS} currentTab={1} menuName={LOCALIZE.ariaLabel.tabMenu} />
+        <TabNavigation
+          tabSpecs={TABS}
+          currentTab={1}
+          menuName={LOCALIZE.ariaLabel.tabMenu}
+          style={{
+            height: "calc(100vh - 241px)",
```

Comment:
This object should be defined in a styles constant.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/152/commits/8a6f6b3854fe17c2ebab4738ef62901e5ddda972